daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/StablePackages.scala

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.language
+
+import com.daml.lf.data.Ref.{PackageId, PackageName}
+
+object StablePackages {
+  // Based on compiler/damlc/tests/src/stable-packages.sh
+  // TODO: Use this map to generate the mapping used in the test,
```

Comment:
What are 'stable' packages?

Reply:
Stable packages are LF packages that are part of the standard library, but don't change with SDK version. They expose Daml types that we want to remain stable across SDK upgrades.